OmiseSDK/Source.swift

```diff
@@ -91,6 +91,7 @@ public struct CreateSourceParameter: Encodable {
 public enum Flow: RawRepresentable, Decodable, Equatable {
     case redirect
     case offline
+    case appredirect
```

Comment:
Since `appredirect` is made of two words, maybe we should call it `appRedirect` like we do with other enums?